style.css

```diff
@@ -353,3 +354,115 @@ li {
   padding: 15px 7px;
   border-style: none;
 }
+
+/* Contact Form */
+
+.c-form {
+  display: flex;
+  flex-direction: column;
+  column-gap: 30px;
+  padding: 20px;
+}
+
+.con-us {
+  color: #9390ff;
+  text-align: center;
+  margin-bottom: 25px;
+  font-size: 30px;
+  font-weight: 500;
+  line-height: 44px;
+  font-family: Helvetica, 'Roboto', 'Ubuntu', sans-serif;
+}
+
+.con-par {
+  color: #f5f5ff;
+  text-align: center;
+  font-size: 16px;
+  font-style: normal;
+  line-height: 23px;
+  padding: 12px;
+  margin-bottom: 23px;
+  font-family: 'Inter', sans-serif;
+}
+
+.li-item {
+  gap: 30px;
+  margin-bottom: 24px;
+}
+
+.name-input,
+.text-input {
+  color: #81818c;
+}
+
+.email-input {
+  color: #191765;
+  border: 1px solid #9490ff;
+}
+
+.detail-input {
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  font-family: 'Inter', Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  background-color: #f7f3ff;
+  width: 100%;
+  margin: 10px 0;
+  border-style: none;
+}
```

Comment:
- [ ] Kindly remove the font-family form placeholder So it matches the Figma design